src/externalized/DefaultContentManager.cc

```diff
@@ -744,17 +744,14 @@ bool DefaultContentManager::loadAmmoTypes()
 
 bool DefaultContentManager::loadMusicModeList(const MusicMode mode, rapidjson::Value &array)
 {
-	std::vector<const ST::string*>* musicModeList = new std::vector<const ST::string*>();
-
 	std::vector<ST::string> utf8_encoded;
 	JsonUtility::parseListStrings(array, utf8_encoded);
 	for (const ST::string &str : utf8_encoded)
 	{
-		musicModeList->push_back(new ST::string(str));
 		SLOGD("Loaded music {}", str);
 	}
 
-	m_musicMap[mode] = musicModeList;
+	m_musicMap[mode] = std::make_unique<decltype(m_musicMap)::mapped_type::element_type>(utf8_encoded);
```

Comment:
Would it be better to store the `utf8_encoded` by value in `m_musicMap` and return a const reference to ST::string in getMusicForMode? Less pointer stuff...